app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,191 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light_gray: #F2F2F2;
+$white: #FFF;
+$beach_blue: #5bc0de;
+
+body {
+  padding-top: 20px;
+}
+
+#index_background_header {
+  text-align: center;
+  padding-top: 300px;
+  font-family: Circular,"Helvetica Neue",Helvetica,Arial,sans-serif;
+  color: #FFF;
+  font-size: 5em;
+  font-weight: bold;
+}
+#search_bar_container .well{
+  background: #000;
+  background: rgba(0,0,0,.1);
+  position: absolute;
+  bottom: 0;
+  margin-bottom: 0;
+  left: 0;
+  right: 0;
+  border-color: transparent;
+}
+#index_image_one {
+  background: url("http://photos.cntraveler.com/2014/07/31/53da72466dec627b149ead94_la-digue-island-seychelles-Arco-images-GmbH-Alamy.jpg");
```

Comment:
URL `http://photos.cntraveler.com/2014/07/31/53da72466dec627b149ead94_la-digue-island-seychelles-Arco-images-GmbH-Alamy.jpg` should not contain protocol or domain